### PR TITLE
docs: changes to docs

### PR DESCRIPTION
**Pull request by paales**

### CODE REVIEW

File: docs/getting-started/vscode.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ as GraphCommerce. Visual Studio Code is free and available on all platforms.
 This guide covers which useful VS Code extensions to install for an optimal
 development experience.
 
-Recommended extensions
+Must have extensions
```
Comment (ErwinOtten) [inline docs/getting-started/vscode.md:16]: Required extensions

Comment (paales) [inline docs/getting-started/vscode.md:16]:
```suggestion
Required extensions
```

File: docs/getting-started/create.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Edit /package.json. Delete `"scripts": {...}` and rename `scripts_local` to
 To order to be able to connect your GraphCommerce app to your Magento backend
 and/or GraphCMS project, you'll need:
 
-- Magento 2.4.3 - Clean install, or a production or development environment
+- Magento 2.4.3 - Clean install, or a production or a development environment
```
Comment (paales) [inline docs/getting-started/create.md:91]:
```suggestion
- Magento 2.4.3 - Clean install, or a production or a development environment
+ Magento 2.4.3 - Clean install, a production or a development environment
```

Comment (paales) [inline docs/getting-started/create.md:91]:
```suggestion
- Magento 2.4.3 - Clean install, a production or a development environment
```

File: docs/getting-started/create.md [hunk 3]
```diff
@@ -145,7 +145,8 @@ the ones you don't want to use.
 Visit the development environment running at http://localhost:3000  
 Visit the GraphQL Playground running at http://localhost:3000/api/graphql
 
-> No success? Refer to [common build errors](../framework/troubleshooting.md)
+> No success? Refer to [common build errors](../framework/troubleshooting.md) or
+> [ask a question on Slack ↗](https://join.slack.com/t/graphcommerce/shared_invite/zt-11rmgq1ad-F~0daNtKcSvtcC4eQRzjeQ)
```
Comment (paales) [inline docs/getting-started/create.md:149]:
```suggestion
> ask your question in the public [Slack community ↗](https://join.slack.com/t/graphcommerce/shared_invite/zt-11rmgq1ad-F~0daNtKcSvtcC4eQRzjeQ) chat
```

**Pull request merged**